frontend/src/components/elements/ArrowVegaLiteChart/ArrowVegaLiteChart.tsx

```diff
@@ -97,7 +97,7 @@ export interface WrappedNamedDataset {
 }
 
 export interface PropsWithHeight extends Props {
-  height: number | undefined
+  height?: number
```

Comment:
This makes `height` truly optional. (Without this change, `<ArrowVegaLiteChart>` and other optionally-height'ed charts, need to be instantiated with `height={undefined}`)